src/NuGet.Clients/NuGet.PackageManagement.UI/Models/LicenseFileText.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel;
+using Microsoft.VisualStudio.Threading;
+using System.Threading.Tasks;
+using NuGet.VisualStudio;
+using System.Globalization;
+using System;
+
+namespace NuGet.PackageManagement.UI
+{
+    internal class LicenseFileText : IText, INotifyPropertyChanged
+    {
+        private string _text;
+        private string _licenseText;
+        private string _licenseHeader;
+        private readonly string _licenseFileLocation;
+        private Func<string, Task<string>> _loadFileFromPackage;
+
+        internal LicenseFileText(string text, string licenseFileHeader, Func<string,Task<string>> loadFileFromPackage, string licenseFileLocation)
+        {
+            _text = text;
+            _licenseHeader = licenseFileHeader;
+            _licenseText = string.Format(CultureInfo.CurrentCulture, Resources.LicenseFile_Loading);
```

Comment:
why use a format if you don't pass any parameters?

Reply:
I have removed it.